user_docs/en/userGuide.t2t

```diff
@@ -2361,19 +2361,24 @@ With several historically popular GUI APIs, the text may be rendered with a tran
 : Default
   Disabled
 : Options
-  Disabled, Enabled
+  Default (Disabled), Enabled, Disabled

```

Comment:
In this case, Default is also missing in "Report live regions ====[BrailleLiveRegions]" and in "==== Speak new text in Windows Terminal via ====[WtStrategy]". Could you add it in this PR? Or is a separate PR required?

Reply:
I've fixed these up now